BUILD.md

```diff
@@ -22,11 +22,27 @@ To compile Cemu, a recent enough compiler and STL with C++20 support is required
 
 For ubuntu and most derivatives:
 
-1) `sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
-2) Run `git clone --recursive https://github.com/cemu-project/Cemu`
-3) `cd Cemu`
-4) `mkdir build && cd build`
-5) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang-12 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-12 -G Ninja`
+
+`sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
+
+For Arch and most derivatives:
+
+`sudo pacman -S cmake git base-devel ninja nasm linux-headers zip spirv-headers`
+
+
+For Fedora and most derivatives:
+
+`sudo dnf in cmake ninja-build vcpkg llvm kernel-headers gtk3-devel libsecret-devel libgcrypt-devel systemd-devel freeglut-devel nasm perl-core zlib-devel cubeb-devel`
+
+1) Run `git clone --recursive https://github.com/cemu-project/Cemu`
+2) `cd Cemu`
+3) `mkdir build && cd build`
+4) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -G Ninja`
+
+4.5) To build for gcc `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++ -G Ninja -DCMAKE_MAKE_PROGRAM=/usr/bin/ninja`
+
 6) `ninja`
 
-Build instructions for other distributions will be added in the future!
+Due to issue with clang and how ubuntu handles it's filesystem and libs some of
+You might not have audio and an option is to use pulseaudio instead of pipewire 
+But this is a very bad idea and could cause more issues with stability. 
```

Comment:
I don't understand what you are referring to when talking about Ubuntu's handling of "filesystem and libs", nor how this paragraph can be useful to users having the issue. It might be more helpful to add `libpulse-dev` to the list of needed dependencies of Debian and Ubuntu, as mentioned in #40.

```suggestion
Due to issue with clang and how Ubuntu handles its filesystem and libs some of
you might not have audio. An option is to use PulseAudio instead of PipeWire,
but this could cause more issues with stability. 
```

@n0toose

> There's a certain amount of typos here (such as its), might be best to cite what you mean using `[text](https://example.com)` and this as a reference: https://wiki.ubuntu.com/MultiarchSpec

Why is Multiarch related to this paragraph?

Reply:
i tried libpulse-dev in fedora and arch and audio still didn't work. if you want to add it to the list make it a PR to my repo i will accept i just don't touch ubuntu. it has caused me enough problems in life.